modules/tinymce/src/plugins/emoticons/main/ts/ui/Dialog.ts

```diff
@@ -98,23 +98,23 @@ const open = (editor: Editor, database: EmojiDatabase): void => {
   dialogApi.focus(patternName);
 
   if (!database.hasLoaded()) {
-    dialogApi.block('Loading emoticons...');
+    dialogApi.block('Loading emojis...');
     database.waitForLoad().then(() => {
       dialogApi.redial(getInitialState());
       updateFilter.throttle(dialogApi);
       dialogApi.focus(patternName);
       dialogApi.unblock();
     }).catch((_err) => {
       dialogApi.redial({
-        title: 'Emoticons',
+        title: 'Emojis',
         body: {
           type: 'panel',
           items: [
             {
               type: 'alertbanner',
               level: 'error',
               icon: 'warning',
-              text: '<p>Could not load emoticons</p>'
+              text: 'Could not load emojis'
```

Comment:
This doesn't appear to ever actually be used? Anyways I removed the `<p>` as per the name this is supposed to be `text` not `html`.

Reply:
For reference this is what it looks like without the `<p>`, so I don't think it should have ever been wrapped anyways:
![image](https://user-images.githubusercontent.com/1575550/146718078-831f5e2d-e03d-4d9d-a087-310cefd40a60.png)